tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/String/StringJoinTests.cs

```diff
@@ -249,8 +249,17 @@ public void GivenATaintedStringInClass_WhenCallingJoin_ResultIsTainted5()
         AssertTaintedFormatWithOriginalCallCheck(":+-tainted-+:", String.Join(",", new ClassForStringTest(taintedValue)), () => String.Join(",", new ClassForStringTest(taintedValue)));
     }
 
-
 #if !NET462
+
+    [Fact]
+    public void GivenATaintedStringInList_WhenCallingJoinWithChar_ResultIsTainted10()
+    {
+        var objectList = new List<object> { TaintedObject, UntaintedObject, OtherTaintedObject };
+        AssertUntaintedWithOriginalCallCheck(
+            () => string.Join(' ', objectList),
+            () => string.Join(' ', objectList));
+    }
+
```

Comment:
Should we add a test case for the new overloads that confirms that the methods are called correctly (and aren't marked tainted)

Reply:
Sure! Added!